### PR TITLE
EHN: add same value count for skew & kurt

### PR DESCRIPTION
**Pull request by auderson**
REF:
* https://github.com/pandas-dev/pandas/pull/46431#issuecomment-1092391777
* https://github.com/pandas-dev/pandas/pull/46431#issuecomment-1093798927

- [ ] closes #xxxx (Replace xxxx with the Github issue number)
- [x] [Tests added and passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#writing-tests) if fixing a bug or adding a new feature
- [x] All [code checks passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit).
- [x] Added an entry in the latest `doc/source/whatsnew/vX.X.X.rst` file if fixing a bug or adding a new feature.

### CODE REVIEW

**Comment by jreback** [comment]
@auderson can you merge master here

**Comment by auderson** [comment]
merged

**Comment by auderson** [comment]
There appears to be a bug in the main branch.

File: pandas/tests/window/test_rolling_skew_kurt.py [hunk 1]
```diff
@@ -219,12 +221,14 @@ def test_rolling_kurt_edge_cases(step):
 
 
 def test_rolling_skew_eq_value_fperr(step):
-    # #18804 all rolling skew for all equal values should return Nan
```
Comment (mroeschke) [inline pandas/tests/window/test_rolling_skew_kurt.py:222]: Could you keep these comments referencing old issues? Helps maintain history of the changed test

**Review by mroeschke — APPROVED** [review]

**Comment by auderson** [comment]
> Thanks again @auderson! (Failure unrelated)

My pleasure

**Pull request merged**